validate/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/coreos/ignition/config"
+	"github.com/coreos/ignition/internal/version"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	flagVersion bool
+	rootCmd     = &cobra.Command{
+		Use:   "ignvalidate [CONFIG.IGN]",
```

Comment:
nit: usually []'s are used to denote optional arguments.
Additionally, this should be updated to include the name of the binary

I'd replace this with `ignition-validate config.ign`